docs/js/getting-started.md

```diff
@@ -20,53 +20,296 @@ import LicenseBadge from '../../src/sap/sdk-js/LicenseBadge'
+
+If you use the Business Partner mock server, enter for `URL` the URL that you have saved from the [previous step](#deployment) and use `NoAuthentication` for `Authentication`. If you use an SAP S/4HANA Cloud system, enter the systems URL in the `URL` field and choose `BasicAuthentication` as authentication type. This will make the fields `User` and `Password` appear. Enter here the credentials of a technical user for your SAP S/4HANA Cloud system.
+
+### Bind destination service
+
+In order to allow the application to use the destination you have just configured, you will need to bind an instance of the destination service and an instance of the `XSUAA service` to your application.
+
+To create an instance of the destination service, execute the following command in your terminal:
+
+```shell
+cf create-service destination lite my-destination
+```
+
+This tells `Cloud Foundry in SAP Cloud Platform` to create an instance of the destination service with service plan `lite` and make it accessible under the name `my-destination`. We can now use the name to bind this service to our application. To do this, open your `manifest.yml` and add a section called `services`, under which you can then add the name of the just created service.
+
+The resulting `manifest.yml` should look like this:
+
+```YAML
+applications:
+  - name: <YOUR-APPLICATION-NAME>
+    path: deployment/
+    buildpacks:
+      - nodejs_buildpack
+    memory: 256M
+    command: node index.js
+    random-route: true
+    services:
+      - my-destination
+```
+
+#### XSUAA Service
+
+Secondly, we need an instance of the `XSUAA service`. The `XSUAA service` is responsible for issuing access tokens that are necessary to talk to other services, like the destination service. For this service, we will need a bit of extra configuration in the form of a configuration file. Create a file called `xs-security.json` with the following content:
+
+```JSON
+{
+  "xsappname": "<YOUR-APPLICATION-NAME>",
+  "tenant-mode": "shared"
+}
+```
+
+The value for `xsappname` again has to be unique across the whole of `Cloud Foundry in SAP Cloud Platform`, so make sure to choose a unique name or prefix.
+
+Now, execute the following command:
+
+```shell
+cf create-service xsuaa application my-xsuaa -c xs-security.json
+```
+
+And, as before, add the newly created services to the services section of your `manifest.yml`.
+
+The final `manifest.yml` should look like this:
+
+```YAML
+applications:
+  - name: <YOUR-APPLICATION-NAME>
+    path: deployment/
+    buildpacks:
+      - nodejs_buildpack
+    memory: 256M
+    command: node index.js
+    random-route: true
+    services:
+      - my-destination
+      - my-xsuaa
+```
+
+Finally, we can replace the parameter of `execute` with an object whose key `destinationName` refers to the name of the destination we defined earlier. If you chose a different name than `Server`, make sure to use it here accordingly.
```

Comment:
Maybe we should also use a different destination name than 'Server'. This is way too general. Maybe something like `S4_SYSTEM` or similar is more appropriate.